nix/hermes.nix

```diff
@@ -0,0 +1,22 @@
+{pkgs ? import ./default.nix {} }:
```

Comment:
https://github.com/crypto-org-chain/chain-main/blob/master/nix/default.nix#L15
I think you can copy the nix script in chain-main for building hermes.

Reply:
i tried, but compiling failed,i'll try again